test/src-tests/index-test.js

```diff
@@ -120,6 +120,10 @@ describe('find-parent/index', function() {
   });
 
   describe('#byClassName', function() {
+    beforeEach(function() {
```

Comment:
Is this needed in this block since none of the tests in this block are using it?